subprojects/packagefiles/glfw/tests/meson.build

```diff
@@ -0,0 +1,49 @@
+###  GLFW test programs
+#@ Build check
+if not get_option('build-tests')
+  subdir_done()
+endif
+
+#@ Constants
+tst_include = include_directories('../include', '../src', '../deps')
+glad_gl = '../deps/glad_gl.c'
+glad_vulkan = '../deps/glad_vulkan.c'
+getopt = '../deps/getopt.c'
+tinycthread = '../deps/tinycthread.c'
+
+#@ Flags
+if is_posix
+  c_flags = ['-D_XOPEN_SOURCE=700']
+endif
```

Comment:
Or this?

Reply:
Same as with the examples, some tests will not build correctly. While GLFW itself is c99, the ancillary programs seem to take a few more liberties.